doc/dev/mgmt/tests.md

```diff
@@ -15,178 +14,136 @@ IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_s
-[This section of the above tutorial](https://docs.microsoft.com/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-describes where to find them
-(besides the subscription ID,
-which is in the "Overview" section of the "Subscriptions" blade.)
+To use the credentials from Python, you need:
+* Application ID (a.k.a. client ID)
+* Authentication key (a.k.a. client secret)
+* Tenant ID
+* Subscription ID from the Azure portal
+[This section of the above tutorial](https://docs.microsoft.com/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key) describes where to find them (besides the subscription ID, which is in the "Overview" section of the "Subscriptions" blade.)
+
+The recommended practice is to store these three values in environment variables called `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, and `AZURE_CLIENT_SECRET`. To set an environment variable use the following commands:
+```Shell
+$env:AZURE_TENANT_ID='<value>'        # PowerShell only
+set AZURE_TENANT_ID=<value>         # Windows CMD (alternatively, use export AZURE_TENANT_ID=<value> to export to the global env)
+export AZURE_TENANT_ID=<value>      # Linux shell only
+```
+*** Note: when setting these variables, do not wrap the value in quotation marks ***
 
 You are now able to log in from Python using OAuth.
 You can test with this code:
 ```python
+import os
 from azure.common.credentials import ServicePrincipalCredentials
 
 credentials = ServicePrincipalCredentials(
-    client_id =  'ABCDEFGH-1234-ABCD-1234-ABCDEFGHIJKL',
-    secret = 'XXXXXXXXXXXXXXXXXXXXXXXX',
-    tenant = 'ABCDEFGH-1234-ABCD-1234-ABCDEFGHIJKL'
+    client_id = os.environ['AZURE_CLIENT_ID'],
+    secret = os.environ['AZURE_CLIENT_SECRET'],
+    tenant = os.environ['AZURE_TENANT_ID']
 )
 ```
 
 ## Providing credentials to the tests
 
-When you run tests in playback mode,
-they use a fake credentials file,
-located at `tools/azure-sdk-tools/devtools_testutils/mgmt_settings_fake.py`,
-to simulate authenticating with Azure.
+When you run tests in playback mode, they use a fake credentials file, located at `tools/azure-sdk-tools/devtools_testutils/mgmt_settings_fake.py`, to simulate authenticating with Azure. In most scenarios you will not have to adjust this file, you will have to make edits to this file if your service uses values that are not already included in the `mgmt_settings_fake.py` file.g
 
-In live mode, you need to use actual credentials
-like those you obtained in the previous section.
-To enable the tests to use them,
-make a copy of the `mgmt_settings_fake.py` file in the same location,
-and rename it `mgmt_settings_real.py`.
+In live mode, you need to use real credentials like those you obtained in the previous section. To enable the tests to use them, make a copy of the `mgmt_settings_fake.py` file in the same location, and rename it `mgmt_settings_real.py`.
```

Comment:
Is that not a contradiction with the guidance to use .env ?

Reply:
Yes, I removed all references to mgmt_settings_real